### PR TITLE
Testing actions and reducers

### PR DESCRIPTION
**Pull request by trbachmann**
Adds testing for all actions, as well as for the notesReducer and errorReducer. Also added file with mock data to use in testing the actions and notesReducer.

### CODE REVIEW

File: src/actions/index.test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import * as actions from './index';
+import * as data from '../mockNotes';
+
+describe('actions', () => {
+  describe('setNotes', () => {
+    it('should return an object with a type of SET_NOTES and notes', () => {
+      const expected = {
+        type: 'SET_NOTES',
+        notes: data.mockNotes
+      };
```
Comment (whitneyfirth) [inline src/actions/index.test.js:10]: Nice work creating our mockData files for this! So handy!

File: src/actions/index.test.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    it('should return an object with a type SET_ERROR of and a message', () => {
+      const mockMessage = 'Note not found';
+      const expected = {
+        type: 'SET_ERROR',
+        message: mockMessage
+      };
+      const result = actions.setError(mockMessage);
+      expect(result).toEqual(expected);
+    });
+  });
```
Comment (whitneyfirth) [inline src/actions/index.test.js:72]: These actions tests look nice and clean, good work!

**Review by whitneyfirth — APPROVED** [review]
Great work on these Tiffany! 😄

File: src/actions/index.test.js (hunk 2 above)
Comment (dForDeveloper) [reply src/actions/index.test.js:72]: 👍

File: src/mockNotes/index.js [hunk 3]
```diff
@@ -0,0 +1,193 @@
+export const mockNotes = [
+  {
```
Comment (dForDeveloper) [inline src/mockNotes/index.js:2]: Good idea making these mockNotes. It keeps the test files a lot cleaner.

**Review by dForDeveloper — APPROVED** [review]
The tests looks good.

**Pull request merged**